src/meta/src/hummock/compaction.rs

```diff
@@ -180,8 +205,14 @@ impl CompactStatus {
 
                 let mut rng = thread_rng();
                 polysst_candidates.shuffle(&mut rng);
```

Comment:
No need to shuffle if sort later?

Reply:
for levels other than L0, this is necessary temporary

Reply:
> No need to shuffle if sort later?

for levels other than L0, this is necessary temporarily

Reply:
Yea I mean maybe add a if-else here.

Reply:
added